lib/features/guide/widgets/about_us/about_us_tab.dart

```diff
@@ -56,30 +56,11 @@ class _AboutUsView extends ConsumerWidget {
                   DescriptionSection(text: value?.aboutUs?.description ?? ''),
                   SectionHeader(text: context.localize.meet_out_team),
                   TeamSection(
-                    members: value?.aboutUsTeam?.whereNonNull
-                            .toList()
-                            .map(
-                              (e) => MemberData(
-                                  socialLinks: e.socialLinks?.whereNonNull
-                                          .toList()
-                                          .map((e) => e.url)
-                                          .toList() ??
-                                      List.empty(),
-                                  name: e.name,
-                                  subtitle: e.subtitle,
-                                  imageUrl: e.photo?.filename_disk ?? ''),
-                            )
-                            .toList() ??
-                        List.empty(),
+                    members: AboutUsDetails().getMemberData(value?.aboutUsTeam),
```

Comment:
When you remove the useless argument from this method you should be able to call 
```dart
value?.aboutUsTeam.getMemberData()
```

or am i wrong? haven't tested it myself